micrometer-java21/src/main/java/io/micrometer/java21/instrument/binder/jdk/VirtualThreadMetrics.java

```diff
@@ -60,18 +68,62 @@ private VirtualThreadMetrics(RecordingConfig config, Iterable<Tag> tags) {
 
     @Override
     public void bindTo(MeterRegistry registry) {
-        Timer pinnedTimer = Timer.builder("jvm.threads.virtual.pinned")
+        Timer pinnedTimer = Timer.builder(METER_NAME_PREFIX + "pinned")
             .description("The duration while the virtual thread was pinned without releasing its platform thread")
             .tags(tags)
             .register(registry);
 
-        Counter submitFailedCounter = Counter.builder("jvm.threads.virtual.submit.failed")
+        Counter submitFailedCounter = Counter.builder(METER_NAME_PREFIX + "submit.failed")
             .description("The number of events when starting or unparking a virtual thread failed")
             .tags(tags)
             .register(registry);
 
         recordingStream.onEvent(PINNED_EVENT, event -> pinnedTimer.record(event.getDuration()));
         recordingStream.onEvent(SUBMIT_FAILED_EVENT, event -> submitFailedCounter.increment());
+
+        bindVirtualThreadSchedulerMXBean(registry);
+    }
+
+    private void bindVirtualThreadSchedulerMXBean(MeterRegistry registry) {
+        try {
+            Class clazz = Class.forName("jdk.management.VirtualThreadSchedulerMXBean");
+            PlatformManagedObject platformMXBean = ManagementFactory.getPlatformMXBean(clazz);
+            Object ignored = clazz.cast(platformMXBean);
+            MethodHandle getParallelism = MethodHandles.publicLookup()
+                .findVirtual(clazz, "getParallelism", MethodType.methodType(int.class));
+            MethodHandle getMountedVirtualThreadCount = MethodHandles.publicLookup()
+                .findVirtual(clazz, "getMountedVirtualThreadCount", MethodType.methodType(int.class));
+            MethodHandle getQueuedVirtualThreadCount = MethodHandles.publicLookup()
+                .findVirtual(clazz, "getQueuedVirtualThreadCount", MethodType.methodType(long.class));
+
+            Gauge.builder(METER_NAME_PREFIX + "parallelism", platformMXBean, o -> invoke(getParallelism, o))
+                .description("Virtual thread scheduler's target parallelism")
+                .register(registry);
+
+            Gauge.builder(METER_NAME_PREFIX + "live", platformMXBean, o -> invoke(getMountedVirtualThreadCount, o))
+                .tag("state", "mounted")
```

Comment:
Since the original version of this PR, I've changed the tag name from `status` to `state`.

Reply:
I've changed it again to `scheduling.status` which I think is far more unambiguous. Let me know if anyone has any thoughts on this.